### ISSUE TITLE
[Bug]: App starts in fullscreen if you used fullscreen before quitting

### ISSUE THREAD

**Issue by mooreye** [FreeTubeApp/FreeTube#4534]
### Guidelines

- [X] I have encountered this bug in the [latest release of FreeTube](https://github.com/FreeTubeApp/FreeTube/releases).
- [X] I have encountered this bug in the [official downloads of FreeTube](https://github.com/FreeTubeApp/FreeTube#official-downloads).
- [X] I have searched the issue tracker for [open](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aopen+is%3Aissue) and [closed](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+is%3Aclosed) issues that are similar to the bug report I want to file, without success.
- [X] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the bug I want to file, without success.
- [X] This issue contains only one bug.

### Describe the bug

1. Watch any video in fullscreen
2. Quit the app with `ctrl+Q` or just closing the window but without quitting from fullscreen first.
3. run app again
4. it will start in fullscreen

### Expected Behavior

app resets to non-fullscreen on start

### Issue Labels

accessibility issue

### FreeTube Version

v0.19.1 Beta

### Operating System Version

fedora 39

### Installation Method

Flathub

### Primary API used

Local API

### Nightly Build

- [ ] I have encountered this bug in the latest [nightly build](https://docs.freetubeapp.io/development/nightly-builds).

**Comment by poltpolt** [comment]
I confirm on steam deck and opensuse

**Comment by jasonhenriquez** [comment]
This is an interesting byproduct of [this feature](https://github.com/FreeTubeApp/FreeTube/pull/2495) being implemented. Don't see an easy fix on this one without altering this feature.

**Comment by absidue** [comment]
We would need some way of determining during shutdown, whether the user entered full screen in the player or already before that. As there are definitely valid uses cases for having the app remembering it's full screen (e.g media centre PCs, when you full screen an app on macOS it gets moved to a separate workspace, so you can still switch back to other windows), but I can completely understand you don't want FreeTube starting in full screen if you only did it to watch a video in full screen.

**Comment by poltpolt** [comment]
on steam deck with no physical keyboard/mouse the only dirty tricky to switch to other windows or even exit is clicking the links in "About" to get firefox open. Otherwise you are stuck with touchscreen, unable to minimise or close.

**Comment by xypage** [comment]
> This is an interesting byproduct of [this feature](https://github.com/FreeTubeApp/FreeTube/pull/2495) being implemented. Don't see an easy fix on this one without altering this feature.

Could we just add an option to disable that entirely then? I imagine most people (me included) don't use full screen while navigating, so there would be no need to remember state since it's something for videos and not the surrounding UI. Maybe an option to default to windowed, full screen, or last used?

**Comment by lothar-cell** [comment]
I actually asked for this feature specifically:D https://github.com/FreeTubeApp/FreeTube/issues/1535
I always use the app in fullscreed and navigate between spaces with gestures. For example Apple Mail and Spotify have this feature built-in, because it is really annoying to press the fullscreen button every time when starting up the app.

**Comment by poltpolt** [comment]
> I actually asked for this feature specifically:D #1535 I always use the app in fullscreed and navigate between spaces with gestures. For example Apple Mail and Spotify have this feature built-in, because it is really annoying to press the fullscreen button every time when starting up the app.

valid point but you have to have an option to quit or minimise the app to regular size - with touchscreen only , there is no way to do it.

**Comment by 2Shot0ccurence** [comment]
For me this problem is actually persistent after a full reinstall.

**Comment by Victor239** [comment]
Is this an irreversible change? I accidentally closed once whilst fullscreen and now it's forever opening in fullscreen.

**Comment by absidue** [comment]
@Victor239 Press F11 to exit full screen, close FreeTube and when you reopen it, it should open normally again.

**Comment by Victor239** [comment]
Thanks! I was disabling fullscreen by through right-clicking the app in my Task Manager (KDE) and unchecking the fullscreen box which didn't change things for next launch, pressing F11 instead worked.

**Comment by poltpolt** [comment]
pressing F11 is not really a solution on keyboardless setups like steamdeck, that is why a button in gui would be appreciated.

**Comment by absidue** [comment]
@poltpolt Pressing F11 was never meant to be a solution, it's a workaround, adding extra buttons that are pointless for most of FreeTube's users, is just another workaround.

If you have a suggestion for an actual solution, please mention it, but repeating the same thing again and again, that isn't a solution, helps nobody.

**Comment by poltpolt** [comment]
I think you underestimate the number of Steamdeck+FreeTube users

**Comment by MrPRAINGLE** [comment]
For whoever still struggling with this: to fix the issue, edit your `settings.db` file. Locate the line of id "bounds", it may be line 7. At the end of this line, you may see that the value of the key "fullScreen" is `true` (if FreeTube always launches in fullscreen mode). Change it to `false` to make it launch in windowed mode. The same can be done backwards in order to make it launch in fullscreen mode.
Hope this helps.
